docs: promote stdlib usage and add migration guide (#46)
Rogdham/pyzstd: docs/pyzstd.rst

=======================
pyzstd module reference
=======================

Introduction
------------

> Pyzstd module provides classes and functions for compressing and decompressing data using `Zstandard <http://www.zstd.net>`_ (or zstd as short
> name) algorithm.

Reviewer: ```suggestion
The pyzstd module provides classes and functions for compressing and decompressing data using `Zstandard <http://www.zstd.net>`_ (or zstd for short) algorithm.
```
